src/main/java/com/aws/greengrass/mqtt/bridge/clients/LocalMqtt5Client.java

```diff
@@ -735,29 +733,29 @@ public void reset() {
             } catch (CrtRuntimeException e) {
                 LOGGER.atWarn().cause(e).log("Unable to start mqtt client, will retry");
                 closeClient();
-                scheduleResetRetry();
+                scheduleResetTask();
             }
 
             // started successfully, reset the reset delay
             resetDelay = DEFAULT_RESET_DELAY;
         }
     }
 
-    private void scheduleResetRetry() {
+    private void scheduleResetTask() {
         synchronized (resetLock) {
-            cancelResetRetry();
-            resetRetryTask = ses.schedule(this::reset, resetDelay.getSeconds(), TimeUnit.SECONDS);
+            cancelResetTask();
+            resetTask = ses.schedule(this::reset, resetDelay.getSeconds(), TimeUnit.SECONDS);
```

Comment:
Use millis and the default delay should be probably 100ms, just to give a bit of time to de-dup requests, but still being quick to reset.